include/rocksdb/perf_level_by_bitfield.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+#include <stdint.h>
+
+namespace rocksdb {
+
+struct PerfLevelByBitField {
+  uint8_t perf_level;
```

Comment:
I think it's possible that by removing all checks of `perf_level`, this field will be no longer needed.

Reply:
then I need to completely rewrite the tests?